### PR TITLE
Refactor quota resources to use the same parseQuotaID

### LINKED ISSUE #1184: Error with network quotas with 1.38
### Terraform Version
version 0.13.5

### Affected Resource(s)
openstack_networking_quota_v2

### Terraform Configuration Files
```hcl
resource "openstack_networking_quota_v2" "quotaset_network" {
  project_id          = openstack_identity_project_v3.project.id
  floatingip          = var.quota_network.floating_ips
  network             = var.quota_network.networks
  port                = var.quota_network.ports
  rbac_policy         = var.quota_network.rbac_policies
  router              = var.quota_network.routers
  security_group      = var.quota_network.security_groups
  security_group_rule = var.quota_network.security_group_rules
  subnet              = var.quota_network.subnets
  subnetpool          = var.quota_network.subnetpools
}

```

### Debug Output
https://gist.github.com/jbe-dw/8a98b52edfa759b68521642e508ca37c

### Expected Behavior
Apply done correctly

### Actual Behavior
Error: Error parsing ID of openstack_networking_quota_v2 11d02006d0814b2fa6aa2ba6bf8177a4: Unable to determine networking quota ID 11d02006d0814b2fa6aa2ba6bf8177a4

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform init`
2. `terraform plan`

### Important Factoids
Run within a CI (docker debian10 image)

### References
None

### PR DESCRIPTION
**Pull request by nikParasyr**
This PR changes lb_quota and networking_quota to use the same func to to parse their resource ID.
Follow up from https://github.com/terraform-provider-openstack/terraform-provider-openstack/pull/1177.

Additionally updateQuotaID was added on util.go with tests, and networking_quota updated to use this also during update
Close #1184  

This will be used as requirement for follow-up PRs for blockstorage and compute quotasets

### CODE REVIEW

**Comment by nikParasyr** [comment]
recheck

**Comment by nikParasyr** [comment]
Changing to draft this and all subsequent due to https://github.com/terraform-provider-openstack/terraform-provider-openstack/issues/1184

File: openstack/resource_openstack_networking_quota_v2.go [hunk 1]
```diff
@@ -185,11 +180,15 @@ func resourceNetworkingQuotaV2Read(d *schema.ResourceData, meta interface{}) err
...
 	config := meta.(*Config)
-	networkingClient, err := config.NetworkingV2Client(GetRegion(d, config))
+	region := GetRegion(d, config)
+	networkingClient, err := config.NetworkingV2Client(region)
 	if err != nil {
 		return fmt.Errorf("Error creating OpenStack networking client: %s", err)
 	}
 
+	// update resource ID with region to allow multi-region quota management
+	updateQuotaID(d, region)
```
Comment (nikParasyr) [inline openstack/resource_openstack_networking_quota_v2.go:190]: updateQuotaID also called in update function to fix #1184. I tested this locally with the failing scenario and it works

**Review by ozerovandrei — APPROVED** [review]
LGTM

**Pull request merged**